### PR TITLE
Add statusUpdatedAt to cluster and host status popover

### PR DESCRIPTION
**Pull request by jtomasek**

### CODE REVIEW

File: src/components/hosts/HostStatus.tsx [hunk 1]
```diff
@@ -80,11 +82,20 @@ const HostStatus: React.FC<HostStatusProps> = ({ host }) => {
     ),
     [status, progressInfo, statusInfo],
   );
+  const footerContent = React.useMemo(
+    () => (
+      <Tooltip content={getHumanizedDateTime(statusUpdatedAt)} position={TooltipPosition.right}>
+        <small>Updated {hdate.relativeTime(statusUpdatedAt || '')}</small>
```
Comment (mareklibra) [inline src/components/hosts/HostStatus.tsx:88]: likewise

**Comment by andybraren** [comment]
Is the output of `getHumanizedDateTime` similar to the timestamps seen in the Events page of the OCP Web Console? On that page it looks like after `10 minutes ago` the format becomes `Jun 2, 5:30pm`, which may be good to align with.

**Comment by jtomasek** [comment]
We're moving from relative time to actual time with this change:
Screenshot 2020-06-03 at 10 09 14

File: src/components/clusters/ClusterStatus.tsx [hunk 2]
```diff
@@ -25,18 +26,28 @@ const getStatusIcon = (status: Cluster['status']) => {
...
-  statusInfo?: string;
+  cluster: Cluster;
 };
 
-const ClusterStatus: React.FC<ClusterStatusProps> = ({ status, statusInfo }) => {
+const ClusterStatus: React.FC<ClusterStatusProps> = ({ cluster }) => {
+  const { status, statusInfo, statusUpdatedAt } = cluster;
   const title = CLUSTER_STATUS_LABELS[status];
   const icon = getStatusIcon(status);
   const headerContent = React.useMemo(() => <div>{title}</div>, [title]);
```
Comment (rawagner) [inline src/components/clusters/ClusterStatus.tsx:36]: IMO all these useMemos are not needed and i'd say its actually _harmful_ to the app performance. These are simple values which do not need to be memoized.

Comment (jtomasek) [reply src/components/clusters/ClusterStatus.tsx]: done

**Pull request merged**